R/CohortCharacterizationDiagnostics.R

```diff
@@ -39,19 +40,25 @@ getCohortCharacteristics <- function(connectionDetails = NULL,
         end
       ))
     }
-    featureExtractionOutput <-
-      FeatureExtraction::getDbCovariateData(
-        connection = connection,
-        oracleTempSchema = tempEmulationSchema,
-        cdmDatabaseSchema = cdmDatabaseSchema,
-        cohortDatabaseSchema = cohortDatabaseSchema,
-        cdmVersion = cdmVersion,
-        cohortTable = cohortTable,
-        cohortId = cohortIds[start:end],
-        covariateSettings = covariateSettings,
-        aggregated = TRUE
-      )
-
+    timeExecution(
+      exportFolder,
```

Comment:
since HADES convention is to use named arguement - maybe this one was missed?

https://ohdsi.github.io/Hades/codeStyle.html#Use_named_arguments